src/test/skript/tests/regressions/1860-Extra String Utilities.sk

```diff
@@ -0,0 +1,16 @@
+test "string cases":
```

Comment:
Don't use a regression test when adding new features. Use syntax test instead. This could be syntaxes/expressions/ExprStringCase.

Reply:
Done!